meson.build

```diff
@@ -751,8 +752,10 @@ exes += executable(
   link_with :    lib_common,
   dependencies : chfn_chsh_deps,
   install_dir : usrbin_exec_dir,
-  install : get_option('enable-chfn-chsh'))
+  install : opt,
+  build_by_default : opt)
```

Comment:
Hmm, is this a bug in meson? With meson-0.54.0-1.fc32.noarch this makes a difference, even though the docs say that `build_by_default` inherits from `install`. @lucabocassi, you touched that code in meson. Am I misunderstanding how this is supposed to work?

Reply:
This was clarified in https://github.com/mesonbuild/meson/issues/7106.